mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/overview/index.tsx

```diff
@@ -0,0 +1,149 @@
+  }
+
+  const newValue = current > previous ? current - previous : previous - current;
+  return (newValue / previous) * 100;
+}
+
+function getWooCommerceTotal(
+  type: 'revenue' | 'orders',
+  period: 'current' | 'previous' = 'current',
+): number | undefined {
+  const overview = select(storeName).getSection('overview');
+  if (overview.data === undefined) {
+    return undefined;
+  }
+
+  const data = overview.data[type] ?? null;
+  if (!data || !data[period]) {
+    return 0;
+  }
+
+  return data[period] as number;
+}
+
+function getWooCommerceDelta(type: 'revenue' | 'orders'): number | undefined {
+  const current = getWooCommerceTotal(type, 'current');
+  const previous = getWooCommerceTotal(type, 'previous');
+
+  if (current === undefined || previous === undefined) {
+    return undefined;
+  }
+  const newValue = current > previous ? current - previous : previous - current;
+  return (newValue / previous) * 100;
+}
+
+export function Overview(): JSX.Element | null {
+  const { overview } = useSelect((s) => ({
+    overview: s(storeName).getSection('overview'),
+  })) as { overview: OverviewSection };
+
+  const percentageFormatter = new Intl.NumberFormat(locale.toString(), {
+    style: 'percent',
+    maximumFractionDigits: 2,
+  });
+  const numberFormatter = new Intl.NumberFormat(locale.toString());
+  const items: JSX.Element[] = [];
+  if (overview.data !== undefined) {
+    items.push(
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      <SummaryNumber
+        key="overview-opened"
+        label={__('Opened', 'mailpoet')}
+        value={`${percentageFormatter.format(getEmailPercentage('opened'))}`}
+        delta={getEmailDelta('opened').toFixed(2) as unknown as number}
+      />,
+    );
+    items.push(
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      <SummaryNumber
+        key="overview-clicked"
+        label={__('Clicked', 'mailpoet')}
+        value={percentageFormatter.format(getEmailPercentage('clicked'))}
+        delta={getEmailDelta('clicked').toFixed(2) as unknown as number}
+      />,
+    );
+  }
+  if (overview.data !== undefined && MailPoet.isWoocommerceActive) {
+    items.push(
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      <SummaryNumber
+        key="overview-orders"
+        label={_x('Orders', 'WooCommerce orders', 'mailpoet')}
+        delta={getWooCommerceDelta('orders').toFixed(2) as unknown as number}
+        value={numberFormatter.format(getWooCommerceTotal('orders'))}
+      />,
+    );
+    items.push(
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      <SummaryNumber
+        key="overview-revenue"
+        label={__('Revenue', 'mailpoet')}
+        delta={getWooCommerceDelta('revenue').toFixed(2) as unknown as number}
+        value={overview.data?.revenue_formatted?.current}
+      />,
+    );
```

Comment:
I see kind of strange percentages here. It shows `%f%%` for orders and revenue. It also looks different from Figma, but I guess maybe that was somewhere discussed that it is OK?

<img width="1495" alt="Screenshot 2023-06-01 at 17 23 55" src="https://github.com/mailpoet/mailpoet/assets/141436/d3d9daeb-0b57-418d-9f3b-a8e6327d0a0c">

Reply:
Should be solved here now: https://github.com/mailpoet/mailpoet/blob/MAILPOET-5088-analytics-page/mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/overview/index.tsx#L70-L72